fineract-provider/src/main/java/org/apache/fineract/portfolio/transfer/service/TransferWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -83,7 +80,6 @@ public class TransferWritePlatformServiceJpaRepositoryImpl implements TransferWr
     private final ClientTransferDetailsRepositoryWrapper clientTransferDetailsRepositoryWrapper;
     private final PlatformSecurityContext context;
 
-    @Autowired
     public TransferWritePlatformServiceJpaRepositoryImpl(final ClientRepositoryWrapper clientRepositoryWrapper,
```

Comment:
Please use Lombok annotations to get rid of the constructor. Let's clean up when we have the opportunity.

Reply:
done